chains/v3/chains_dev.json

```diff
@@ -726,6 +726,45 @@
                     "existentialDeposit": "100000000000000000",
                     "transfersEnabled": true
                 }
+            },
+            {
+                "assetId": 24,
+                "symbol": "GENS",
```

Comment:
Should we add the GENS price @antonkhvorov ?

https://www.coingecko.com/ru/%D0%9A%D1%80%D0%B8%D0%BF%D1%82%D0%BE%D0%B2%D0%B0%D0%BB%D1%8E%D1%82%D1%8B/genshiro

Reply:
Yes, I think it would be good

Reply:
done